docs/manuals/bpmn-guide/README.md

```diff
@@ -260,8 +260,9 @@ Example:
       ]
     }
 
-
 ```
+:warning: prefixing the reference table with 'BPMN_' is recommended to avoid conflicts with other ZAAK types and reference tables.
```

Comment:
```suggestion
:warning: prefixing the reference table code with 'BPMN_' is recommended to avoid conflicts with other ZAAK types and reference tables.
```